superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -139,20 +137,20 @@ def _train_step(self, precision):
         curr_step = 0
         while True:
             for idx, sample in enumerate(self._dataloader):
-                start = time.time()
+                start = self.timer()
                 if self._gpu_available:
                     sample = sample.cuda()
                 self._optimizer.zero_grad()
                 output = self._model(sample)
                 loss = self._loss_fn(output, self._target)
                 loss.backward()
                 self._optimizer.step()
-                end = time.time()
+                end = self.timer()
                 curr_step += 1
                 if curr_step > self._args.num_warmup:
                     # Save the step time of every training/inference step, unit is millisecond.
                     duration.append((end - start) * 1000)
-                if self._is_finished(curr_step, end):
+                if self._is_finished(curr_step, end, 100):
```

Comment:
When we change the code, we will replace all 100. It's really hard to find all.

Reply:
updated to use a variable